srv/salt/ceph/stage/cephfs/default.sls

```diff
@@ -1,4 +1,4 @@
-
+{% if salt.saltutil.runner('select.minions', cluster='ceph', roles='mds') != [] %}
```

Comment:
Jinja statements here are interpreted as Python, yes? In such a case, `if not salt.saltutil.runner('select.minions', cluster='ceph', roles='mds')` is the proper 'Pythonic' way to test for an empty list.